whipper/test/test_common_path.py

```diff
@@ -9,22 +9,28 @@
 class FilterTestCase(common.TestCase):
 
     def setUp(self):
-        self._filter = path.PathFilter(special=True)
+        self._filter_default = path.PathFilter()
```

Comment:
Isn't this the same as `self._filter_posix`? The default is "`posix=True, fat=False`", no? Maybe it'd be better to have a test `self._filter_none = path.PathFilter(posix=False, fat=False)`?